oc-aliases.sh

```diff
@@ -2,75 +2,82 @@
 # Licensed Materials - Property of IBM
 # 5737-I32
 #
-# (C) Copyright IBM Corp. 2021
+# (C) Copyright IBM Corp. 2022
 #
 # US Government Users Restricted Rights - Use, duplication, or
 # disclosure restricted by GSA ADP Schedule Contract with IBM Corp.
 ########################################################## {COPYRIGHT-END} ###
 
+if [[ -z "${PROJECT_NAME}" ]]; then
+  echo -e "$(date +%Y-%m-%d\ %H:%M:%S) - \033[0;31mERROR\033[0m - Environment variable PROJECT_NAME is not defined. Try export PROJECT_NAME=\"<deployment_namespace>\""
+fi
+
 function podlog {
-	podlabel=$1
-	oc logs $(oc get pods -n=spectrum-discover -l app=${podlabel} -o=jsonpath='{.items[0].metadata.name}') -n=spectrum-discover
+	selector=$1
+	oc -n ${PROJECT_NAME} logs --selector=${selector} --all-containers
```

Comment:
I would prefer if we also add "-l app=isd" here when querying the logs

Reply:
I think that is an unnecessary additional overhead